Index.html

```diff
@@ -4,281 +4,379 @@
+                  sign-ups
+                  required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                  took
+                  a standard dummy text.</p>
+              <ul class="web-icons icons-desktop">
+                  <li>
+                      <img alt="Web Icon" class="icon-web" src="images/Iconic /css.png">
+                  </li>
+                  <li>
+                      <img alt="Web Icon" class="icon-web" src="images/Iconic /html.png">
+                  </li>
+                  <li>
+                      <img alt="Web Icon" class="icon-web" src="images/Iconic /bootstrap.png">
+                  </li>
+                  <li>
+                      <img alt="Web Icon" class="icon-web" src="images/Iconic /puby.png">
+                  </li>
+              </ul>
+              <form>
+                  <button type="submit" class="animate-btn project-desktop-button button1 button1-text sub-1">See
+                      Project</button>
+              </form>
           </div>
-          <div class="pro-art">
-            <h2 class="headline-4">Profesional Art Printing Data</h2>
-            <p class="text-content-2">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. Has been the industry's standard.
-            </p>
-            <ul class="btn-list-1">
-              <li>
-                <button class="btn-html-1"><a href="#">HTML</a></button>
-              </li>
-              <li>
-                <button class="btn-bootstrap-1">
-                  <a href="#">BOOTSTRAP</a>
-                </button>
-              </li>
-              <li>
-                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
-              </li>
-            </ul>
-            <button class="btn-sub-2"><a href="#">See project</a></button>
+      </div>
+      <div class="projects-section2">
+          <div class="page2">
+              <h2 class="heading-two heading4 text-color h2-margin">Profesional Art Printing Data</h2>
+              <p class="project-discription project-discription2 text-color">A daily selection of privately
+                  personalized reads; no accounts
+                  or
+                  sign-ups required. has been the industry's standard</p>
+              <ul class="web-icons web-icons4">
+                  <li>
+                      <button type="submit" class="web-icon-page2 icon-web">html</button>
+                  </li>
+                  <li>
+                      <button type="submit" class="web-icon-page2 icon-web">bootstrap</button>
+                  </li>
+                  <li>
+                      <button type="submit" class="web-icon-page2 icon-web">Ruby</button>
+                  </li>
+              </ul>
+              <button type="submit" class="animate-btn button4 button2 button1-text">See Project</button>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly consider adjusting headings and `li` tags in these projects' cards to make sure they look identical to the figma design. I checked yours and they still are not in the same position as they are in the figma.
your design
![q](https://github.com/MohammadYaser/Portfolio/assets/96683843/c0b06885-0d13-45f4-9920-e63737b725de)

figma design
![as](https://github.com/MohammadYaser/Portfolio/assets/96683843/dad0ceac-0d3a-4b19-b6ec-07b925b53d74)

- [OPTIONAL] Also kindly consider adding the hover styling to these projects' cards. this requirement was missed by the previous reviewer. so the requirement is: when the user hovers over these cards, then the cards' description disappears and a red `see project` button appears as shown in the figma design.
![r](https://github.com/MohammadYaser/Portfolio/assets/96683843/88456273-0bd8-4e41-a16d-f809f504e319)